spec/models/conference_spec.rb

```diff
@@ -778,6 +778,33 @@
     end
   end
 
+  describe '#scheduled_event_distribution' do
+    let!(:conference) { create(:conference) }
```

Comment:
@Ana06 yes sorry my bad, I was thinking that I will have to call conference specifically in both the test. Yes you are right it will be called automatically just before `confirmed_unscheduled_event` because we are using conference in it. I have made the changes. :slightly_smiling_face: